packages/dev/loaders/src/OBJ/solidParser.ts

```diff
@@ -275,6 +276,17 @@ export class SolidParser {
         //Pattern5 => triangle = ["-1/-1/-1","-2/-2/-2","-3/-3/-3","-1/-1/-1","-3/-3/-3","-4/-4/-4"];
     }
 
+    /**
+     * To get color between color and extension color
+     * @param index Integer The index of the element in the array
+     * @returns value of target color
+     */
+    private _getColor(index: number) {
+        if (!this._loadingOptions.importVertexColors) return undefined;
+        if (this._extColors.length > 0) return this._extColors[index];
+        return this._colors[index];

```

Comment:
no need to actually test for length > 0
```suggestion
        if (this._loadingOptions.importVertexColors) {
            return this._extColors[index] ?? this._colors[index];
        }
```